src/viewer.js

```diff
@@ -364,6 +366,7 @@ $.Viewer = function( options ) {
         viewer: this
     });
 
+    var self = this; // Capture viewer instance
```

Comment:
We already have one of these at the very top of the function; it's called `_this`. Better to use that than to add a second one.